src/dex.hs

```diff
@@ -37,6 +41,29 @@ data EvalMode = ReplMode String
 
 data CmdOpts = CmdOpts EvalMode (Maybe FilePath) EvalConfig
 
+
+
+filteredCompletions candidates str = map simpleCompletion $
+  filter (str `isPrefixOf`) candidates
+
+completions :: CompletionFunc (StateT TopEnv IO)
+completions (line, _) = do
+  env <- get
+  let varNames = map (show . pretty) $ envNames env
```

Comment:
`(show . pretty)` is almost certainly not the right way to get the names as strings.
idk what is though

Reply:
`pprint` would do the trick here.